layer/contracts/runs.py

```diff
@@ -38,6 +34,13 @@ def local_relative_paths(self) -> Iterator[str]:
                     yield os.path.relpath(dir_file_path)
 
 
+@enum.unique
+class TaskType(enum.IntEnum):
+    INVALID = 0
```

Comment:
why keep the invalid? let's throw when we cannot map it should be caught in E2E tests

Reply:
agreed, fixed (in https://github.com/layerai/sdk/pull/414)